sdk/core/core-lro/test/utils/testOperation.ts

```diff
@@ -31,16 +35,16 @@ async function update(
   const doFinalResponse = previousResponse && previousResponse.parsedBody.doFinalResponse;
 
   if (!initialResponse) {
-    response = await client.sendInitialRequest(new TestWebResource(abortSignal));
+    response = await client!.sendInitialRequest(new TestWebResource(abortSignal));
```

Comment:
I know it's test code, but if client is optional, shouldn't this have an if check around it?

Reply:
I'm going to throw if client doesn't exist a bit above this code. I'm also adding a comment explaining why client is possibly undefined (even if most certainly always defined).